src/SFA.DAS.LevyTransferMatching.Web/Models/Pledges/ConfirmationViewModel.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.LevyTransferMatching.Web.Models.Pledges
+{
+    public class ConfirmationViewModel
+    {
+        public string EncodedAccountId { get; set; }
+        public string PledgeId { get; set; }
```

Comment:
As this will be the encoded PledgeId we should probably name it accordingly.

Reply:
Will change the name